jx_sqlite/aggs_table.py

```diff
@@ -360,12 +360,21 @@ def _get(row):
                     column_number = len(outer_selects)
 
                     for json_type, sql in details.sql.items():
-                        concat_sql.append("GROUP_CONCAT(QUOTE(DISTINCT(" + sql + ")))")
+                        concat_sql.append("GROUP_CONCAT(DISTINCT(" + sql + "))")
```

Comment:
Don't know why, but in my local branch test_union_multiple fails. Tried changes as per your suggestions.

```
ERROR: {
    "data": {
        "a": ["x", "y"],
        "c": ["x", "y"]
    },
    "meta": {"format": "cube"},
    "select": [{"name": "c"}, {"name": "a"}]
} does not match expected {
    "data": {
        "a": ["x", "y"],
        "c": [2, 3, 4, 5, 6, 7, 8]
    },
    "meta": {"format": "cube"}
}
```

Also I am confused for why the output of both "a" and "c" is same and wrong.